plugins/modules/site_workflow_manager.py

```diff
@@ -410,7 +410,7 @@
 """
 
 floor_plan = {
-    '101101': 'Cubes And Walled Offices',
+    '107107': 'Cubes And Walled Offices',
```

Comment:
do we need these changes? Have you got a chance to discuss with Abhishek on this? 
If you do these changes, will it work for all cases? Is it backward compatible?